esignet-mock.properties

```diff
@@ -142,8 +145,8 @@ mosip.esignet.mock.vciplugin.verification-method=${mosip.esignet.vci.authn.jwk-s
 mosip.esignet.supported.authorize.scopes={'Manage-Identity-Data','Manage-VID','Manage-Authentication','Manage-Service-Requests','Manage-Credentials'}
 mosip.esignet.supported.openid.scopes={'profile','email','phone'}
 mosip.esignet.openid.scope.claims={'profile' : {'name','address','gender','birthdate','picture','email','phone_number'},'email' : {'email'}, 'phone' : {'phone_number'}}
-mosip.esignet.supported.credential.scopes={'mock_identity_vc_ldp'}
-mosip.esignet.credential.scope-resource-mapping={'mock_identity_vc_ldp' : '${mosip.esignet.domain.url}${server.servlet.path}/vci/credential'}
+mmosip.esignet.supported.credential.scopes={'mock_identity_vc_ldp', 'mosip_identity_vc_ldp'}
```

Comment:
correct the key name and as its mock properties file we do not have to list mosip_identity_vc_ldp